tests/fuzz/runFuzzTest.sh

```diff
@@ -54,9 +54,9 @@ FUZZCOV_SOURCES="${S2N_ROOT}/api ${S2N_ROOT}/bin ${S2N_ROOT}/crypto ${S2N_ROOT}/
 
 if [ -e $TEST_SPECIFIC_OVERRIDES ];
 then
-    export LD_PRELOAD="$TEST_SPECIFIC_OVERRIDES $GLOBAL_OVERRIDES"
+    export LD_PRELOAD_="$TEST_SPECIFIC_OVERRIDES $GLOBAL_OVERRIDES"
```

Comment:
Probably useful to include a comment about why we're doing the `_` shenanigans.

Reply:
Added a comment to explain this.